### PR TITLE
added few api for grocery websites

### PR DESCRIPTION
**Pull request by VinothKumarBE**
I have done a few Api for grocery websites using Node.Js.  please review it

### CODE REVIEW

File: starter/app.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+//  const Server =http.createServer((req, res) => {
+//     console.log(req)
+//     res.end('hello welcome to the server side');
+
+// });
+// Server.listen('8900', '127.0.0.1', () =>{
+//     console.log('Listnening to request on port  8000')
+// })
+
+
```
Comment (MahaBlockoville) [inline starter/app.js:49]: Don't commit commented code in pr

Comment (MahaBlockoville) [reply starter/app.js:49]: I don't think this is a correct flow. kindly refer these github projects, you may follow Udemy course but also refer this as a example

https://github.com/nnance/express-ejs-html
https://github.com/maitraysuthar/rest-api-nodejs-mongodb

**Comment by VinothKumarBE** [comment]
Okay, Mam. I take that link and refer to that mam. Thank you.

________________________________

File: starter/app.js (hunk 1 above)
Comment (VinothKumarBE) [reply starter/app.js:49]: > Don't commit commented code in pr


Sure. Hereafter, I will clean up commented code before check-in.

**Pull request merged**